docs/application/native/guides/device/system.md

```diff
@@ -496,6 +498,20 @@ The following table lists the OpenGL&reg; ES feature keys.
 | `http://tizen.org/feature/opengles.version.3_1` | `bool`   | The platform returns `true` for this key and the `http://tizen.org/feature/opengles` key, if the device supports the OpenGL&reg; ES version 3.1. |
```

Comment:
- Information on Web and .Net seems different as per the information given here, for native. This could be due to the different technology platform whereas its better to confirm for the same.

- At line no. 24 "Table: System information keys" is missing

- In 'Feature Keys' tables "Attach Panel Feature Keys" table is mentioned here but not on the live page, below is the code for the same:
---- From Line No. 209
 The following table lists the attach panel feature keys.
 
 <a name="attachpanel"></a>
 **Table: Attach panel feature keys**
 
 | Key                                     | Type   | Description                              |
 |-----------------------------------------|--------|------------------------------------------|
 | `http://tizen.org/feature/attach_panel` | `bool` | The platform returns `true` for this key, if the device supports the attach panel feature. |
----- to Line No. 217